src/scss/components/_label.scss

```diff
@@ -0,0 +1,12 @@
+.udlite-form-label {
```

Comment:
Remove udlite prefix.
Change the name of this file to _form.scss
.form {
      &-label {
         display: flex;
         align-items: center;
         margin: 0;
        padding-bottom: $padding-bottom--8;
        min-height: $min-height--28;
        position: absolute;
        height: $height--1;
        width: $width--1;
        overflow: hidden;
        clip: $form-label-clip;
      }
}